.github/workflows/doctor.yml

```diff
@@ -28,4 +28,8 @@ jobs:
 
       - name: Check for zero diffs
         id: diff
-        run: gromit git diff ${{ matrix.repo }}
+        run: |
+          output=$(gromit git --loglevel trace diff ${{ matrix.repo }} 2>&1)
```

Comment:
Is trace level needed here?

Reply:
Not really, I would output this to the info filter.